metricflow/model/transformations/proxy_measure.py

```diff
@@ -27,24 +27,24 @@ def transform_model(model: UserConfiguredModel) -> UserConfiguredModel:
 
                 add_metric = True
                 for metric in model.metrics:
-                    if metric.name == measure.name:
+                    if metric.name == measure.reference:
                         if metric.type != MetricType.MEASURE_PROXY:
                             raise ModelTransformError(
-                                f"Cannot have metric with the same name as a measure ({measure.name}) that is not a "
+                                f"Cannot have metric with the same name as a measure ({measure.reference}) that is not a "
```

Comment:
Similarly, is printing `measure.reference.element_name` preferred?

Reply:
Updated.